airflow/utils/helpers.py

```diff
@@ -154,19 +154,32 @@ def as_flattened_list(iterable):
 
 
 def chain(*tasks):
-    """
-    Given a number of tasks, builds a dependency chain.
+    r"""
+    Given a number of tasks or list of tasks, builds a dependency chain.
 
-    chain(task_1, task_2, task_3, task_4)
+    chain(task_1, [task_2, task_3], task_4, task_5)
 
     is equivalent to
 
+           / --> task_2 \
+    task_1               --> task_4 --> task_5
+           \ --> task_3 /
+
     task_1.set_downstream(task_2)
-    task_2.set_downstream(task_3)
+    task_1.set_downstream(task_3)
+    task_2.set_downstream(task_4)
     task_3.set_downstream(task_4)
+    task_4.set_downstream(task_5)
+
+    :param tasks: tasks or list of tasks
+    :type tasks: airflow.models.BaseOperator
     """
     for up_task, down_task in zip(tasks[:-1], tasks[1:]):
-        up_task.set_downstream(down_task)
+        if isinstance(up_task, list):
```

Comment:
Is this necessary? According to the documentation for the `set_downstream` method, the parameter can be both one task and a task list.
http://airflow.apache.org/code.html#airflow.models.BaseOperator.set_downstream